verify-fabstracts.sh

```diff
@@ -1 +1,4 @@
+#!/bin/bash
+set -xev
+lean/build/release/shell/lean --version
```

Comment:
When lean is built, it should install an executable at `lean/bin/`. If I remember right, using the one in the build directory (as you do here) leads to trouble finding the standard library.